frappe/__init__.py

```diff
@@ -52,8 +52,30 @@ def _(msg, lang=None):
 	if lang == "en":
 		return msg
 
-	return get_full_dict(local.lang).get(msg) or msg
+	# To parse html data
+	msg = parse_msg(msg)
```

Comment:
For example, user created below translations

![translation](https://cloud.githubusercontent.com/assets/8780500/13388531/b2e67a26-dee6-11e5-8e4a-73a55c1d76bb.png)

Now user has item with below description
Intel Core i5 processor
Intel HD Graphics 6000
Up to 12 hours of battery life1 (13-inch model)
802.11ac Wi-Fi

So whenever user select the language as Italiano in print format, system has to translate the description into user's custom translation language as per attached image. But system can not translated, so after debug I understand that description has html data like

(div) (font color="#111111") Intel Core i5 processor' (/font) (/div)
(div) (font color="#111111") Intel HD Graphics 6000 (/font)(/div)
(div)(font color="#111111")Up to 12 hours of battery life1 (13-inch model)  (/font)(/div)
(div)(font color="#111111")802.11ac Wi-Fi (/font)(/div)

Note: for preview I have used '(' instead of '<' and ')' instead of '>' in above example.

 
During translation the structure is as below

{u'Intel Core i5 processor\nIntel HD Graphics 6000\nUp to 12 hours of battery life1 (13-inch model)\n802.11ac Wi-Fi' : u'processore Intel Core i5\nIntel HD Graphics 6000\nFino a 12 ore di life1 batteria ( 13 pollici modello)\n802.11ac Wi - Fi'}

So in dictionary the data has '\n' for new line while in description it come up as html data.

Therefore if data has html content then system makes the below flow
parse html and get data -> translate data into custom translations -> again make data as html